apps/studio/src/components/PageEditor/FileAttachment.tsx

```diff
@@ -23,13 +24,14 @@ export const FileAttachment = ({
   siteId,
   maxSizeInBytes,
   acceptedFileTypes,
+  shouldFetchResource = true,
```

Comment:
curious - seem like default is `true` but the only place that uses this component hardcode it to be `false` - not too sure what's the intention of this props and when/why to use it

Reply:
there are 2 places that use this component - `LinkEditor`, where it is set to `false` and `JsonFormsImageControl`, where it is `true`!